From 4c4d4a7f58397e779896b9fc47857a9a9319b19b Mon Sep 17 00:00:00 2001
From: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Date: Thu, 13 Jun 2024 13:54:43 +0200
Subject: [PATCH 1133/1695] dts: stm32: change default d3 domain source clock
 on stm32mp257f-ev1

Change the default source clock to MSI kernel clock.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c0f2f9a799abe61ab1f71eb6fa92c43af6162a1e dts: stm32: change default d3 domain source clock on stm32mp257f-ev1]
Change-Id: I817c9d01dff95de2f99648a6b16c1bfebbf7ec4c
Signed-off-by: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/387757
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi b/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
index db8827f6a..8c9314fed 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
+++ b/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
@@ -109,7 +109,7 @@
 		MUX_CFG(MUX_LVDSPHY, MUX_LVDSPHY_FLEX32)
 		MUX_CFG(MUX_DTS, MUX_DTS_HSE)
 		MUX_CFG(MUX_RTC, MUX_RTC_LSE)
-		MUX_CFG(MUX_D3PER, MUX_D3PER_LSE)
+		MUX_CFG(MUX_D3PER, MUX_D3PER_MSI)
 		MCO_CFG(MCO1, MUX_MCO1_FLEX61, MCO_OFF)
 		MCO_CFG(MCO2, MUX_MCO2_FLEX62, MCO_OFF)
 	>;
-- 
2.47.2

